core/coreunix/add.go

```diff
@@ -390,7 +389,12 @@ func (adder *Adder) addNode(node node.Node, path string) error {
 	}
 	dir := gopath.Dir(path)
 	if dir != "." {
-		if err := mfs.Mkdir(mr, dir, true, false); err != nil {
+		opts := mfs.MkdirOpts{
+			Mkparents: true,
+			Flush:     false,
+			Prefix:    adder.Prefix,
+		}
+		if err := mfs.Mkdir(mr, dir, opts); err != nil {
```

Comment:
I think the argument to mkdir should be a pointer so we can pass `nil` and have that assume default values.

Reply:
I'm not sure what the defaults should be in that case.